index.html

```diff
@@ -8,66 +8,73 @@
+              <ul class='social media'>
+                <a href='https://www.facebook.com/lacelin2010'><img src='imgs/facebook.png'></a>
+                <a href='https://twitter.com/LaceyVu'><img src='imgs/twitter.png'></a>
+                <a href='https://www.linkedin.com/profile/view?id=AAMAABFJIagBqvNwtZ8994EGUah_0nwhuXFxaRA&trk=hp-identity-photo'><img src='imgs/linkedin.png'></a>
+                <a href='https://github.com/laceylin2010'><img src='imgs/github.jpg'></a>
+              </ul>
+              <ul class='menubar'>
+                <img src='imgs/hamburger.png' class='hamburger'>
+                <li id='mainpage'>Home</li>
+                <li id="about-me">About</li>
+                <li class='projectlink' id="projecthistory" >Projects</a></li>
+                <li class='employed' id='employedhistory'>Employment</li>
+              </ul>
+          </nav>
+      </header>
     </section>
-  </section>
-    <section id='portfolio'>
-      <article class='template' data-category='projects'>
-        <header>
-            <div class='pers-proj'>
-              <h2><a href=''>Title</a></h2>
-              By: <span class='authorname'> Author Name </span><br></br>
-              Published <time class='datetime' datePublished='2000-01-01'>Published Time </time>
-            </div>
-        </header>
-      </article>
-    </section>
-    <section id='employment' class='jobhistory'>
-      <article class='template2' data-category='previousjobs'>
-          <header>
-            <h2>Job Title</h2>
-              <p class='company'>Company</p>
-              <p class='employeddates'>Dates Worked</p>
-              <p class='jobdescription'>Job Description</p>
+    <section class= 'seven columns'>
+      <section id='aboutme' class='bio' >
+        <h1>About Me</h1>
+          <p>My name is Lacey and I have an Associates in Business from Highline Community College. Later down the years, I found out that I did not know what to exactly do with my degree and ended up working a bunch of jobs that did not relate as much instead. I heard of code fellows from a friend and decided to give it a try. After taking 201 I have realized how much I enjoy it and how awesome it would be to turn into a career. So here goes!</p>
+      </section>
+      <script id='testing' type='text/x-handlebars-template'>
+        <section id='portfolio'>
+          <article class='template' data-category='projects'>
+            <header>
+              <div class='pers-proj'>
+                <h2><a href='{{titleUrl}}'>{{title}}</a></h2>
+                <p>By: <span class='authorname'> {{author}} </span><br></br>
+                Published <time class='{{publishedOn}}' datePublished='2000-01-01'>Published Time </time>
+              </div>
             </header>
-        </article>
+          </article>
+        </section>
       </section>
+      </script>
+      <div id="myCoolStuff"></div>
 
+      <script id='testing2' type='text/x-handlebars-template'>
```

Comment:
@laceylin2010 be careful, when naming things with a number based convention - like "template1" or "template2" - this can cause confusion and it's best to name your items what they are - example:  "employment-template" / "about-me-template"